src/components/PageContent/Avatar/Avatar.jsx

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright (C) 2018 The Trustees of Indiana University
+SPDX-License-Identifier: BSD-3-Clause
+*/
+import classNames from "classnames";
+import * as PropTypes from "prop-types";
+import * as React from "react";
+import * as Rivet from "../../util/Rivet";
+import { parseRivetMultiListUtility } from "../../util/RivetizeAux.js";
+const classPrefix = "rvt-breadcrumbs";
+
+/**
+ * Use the avatar component to display a person’s picture or initials with a rounded border.
+ */
+const Avatar = ({
+  className,
+  children,
+  initials,
+  size,
+  src,
+  ...attrs
+}) => {
+  const classNameArr = [
+    "rvt-avatar"
+  ]
+
+  if (size) {    
+    classNameArr.push(parseRivetMultiListUtility(size, Sizes, 'rvt-avatar-'))
+  }
+
+  if (className) {
+    classNameArr.push(className)
+  }
```

Comment:
I'm not sure this array placeholder is necessary. The `classNames` library can handle null/undefined values pretty well.